tests/unit/src/test/scala/tests/OnTypeFormattingSuite.scala

```diff
@@ -421,6 +422,28 @@ class OnTypeFormattingSuite extends BaseLspSuite("onTypeFormatting") {
     stripMarginEnabled = false
   )
 
+  check(
+    "add-stripMargin-with-config".only,
```

Comment:
Remove `only`?

Reply:
we could probably have a Scalafix linting rule for this. It could be shipped with MUnit